code/modules/antagonists/clock_cult/items/brass_clothing.dm

```diff
@@ -18,10 +18,10 @@
 
 /obj/item/clothing/suit/clockwork/equipped(mob/living/user, slot)
 	. = ..()
-	if(istype(user, /mob/living/carbon/human/consistent) || istype(user, /mob/living/carbon/human/dummy))
+	if((istype(user, /mob/living/carbon/human/consistent) && !user.client) || (istype(user, /mob/living/carbon/human/dummy) && !user.client))
```

Comment:
One would expect dummies and consistent to only exist in unit tests, but something is screwing me and I don't know what, so I think this is the clearest option.

Reply:
Dummies exist for mob previews

Reply:
Regular crew are being accepted as types of consistent or dummy humans

Reply:
That makes no sense, that should never happen

Reply:
I really don't like this, something is definitely going wrong if this is needed and its hiding some other erronous behaviour